### DISCUSSION TITLE
Hyper Forms

### DISCUSSION THREAD

**Discussion by KrabbyCurse** [Techokami/SonicWorldsNext#50] [Polls]
Hello There
I've got my first poll:

**Comment by DimensionWarped** [comment]
Hyper forms aren't really on any kind of roadmap for us right now and they are a bit trivial in concept anyway. I imagine this is something we would probably take on if a pull request were made that already had this pretty much built and we maybe had to change just some small details of how we want to expose any relevant flags, but this discussion board isn't really for just proposing new features regardless of whether they are in the form of wishlists, polls, or what have you.

Perhaps when we have exhausted our current task lists we'll consider an open thread for feature proposals, but I'm going to ask people to refrain from starting threads about them unless they are specifically volunteering to implement those features themselves (and in a timely manner).
